src/apps/bridge/sensor_drivers/rbrCodaSensor.cpp

```diff
@@ -386,11 +389,11 @@ BmRbrDataMsg::SensorType_t RbrCodaSensor::rbrCodaGetSensorType(uint64_t node_id)
               break;
             }
             key_str[len] = '\0';
+            if (cbor_value_advance(&sys_cfg_map) != CborNoError) {
+              printf("Failed to advance sys_cfg_map past string key\n");
+              break;
+            }
             if (strcmp(key_str, "rbrCodaType") == 0) {
-              if (cbor_value_advance(&sys_cfg_map) != CborNoError) {
-                printf("Failed to advance the sys_cfg_map\n");
-                break;
-              }
```

Comment:
This was a 🐛 discovered while validating this change. We were only advancing past the string key in the map when it matched `rbrCodaType`. If that was not the first key, we failed our search for the type.